planning.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Planning an event
+sidebar_link: false
+---
+
+We are always interested in ideas for new events. It could be a presentation night on a particular topic, or a new kind of event altogether.  Many of our current events were first proposed by members.
+
+To start planning an event:
+
+1. Join us in the **#organizing** channel in [our Slack workspace](slack.md).  Start a conversation along the lines of "I'm thinking about XYZ," and get a discussion going about what shape the event could take.
+
+1. Find a [leader](contact.md) to pair with in the planning process. You will need their help.
```

Comment:
Maybe instead of "you will need their help", we should mention the ways in which they will help. eg, making sure the event is right for BP, scheduling it, getting it listed on meetup.com, etc.

Reply:
Also, I wonder if we should designate a few (<=3) named leaders as points of contact - this might be easier to for people to deal with than a list of email addresses, some of whom point to people who are no longer very active.